### PR TITLE
Render oriented crops in viewer beta

### PR DESCRIPTION
**Pull request by kerrj**
Adds support for for rendering with oriented boxes in the new viewer.

image

https://github.com/nerfstudio-project/nerfstudio/assets/15806475/7e35c4e4-e433-44c1-b10e-8b5cea752b52

### CODE REVIEW

File: nerfstudio/viewer_beta/render_panel.py [hunk 1]
```diff
@@ -539,8 +544,100 @@ def _(_) -> None:
     def _(event: viser.GuiEvent) -> None:
         """TODO: write the render JSON and show the render command."""
```
Comment (brentyi) [inline nerfstudio/viewer_beta/render_panel.py:545]: Let's remove the TODO?

**Review by brentyi — COMMENTED** [review]
lgtm, let's fix the formatting?

**Review by brentyi — APPROVED** [review]

**Comment by smart4654154** [comment]
Thank you very much for your work. My question seems to be related to this job. What is the meaning of the crop scale value in the new version of viewer? I found that the value of the crop scale cannot match the coordinate values of the world coordinate system. For example, when the crop scale is set to 4, 4, 4, it is approximately equivalent to a square with a side length of 40 in the world coordinate system
image
What are the meanings of "crash max" and "crash min" in Viewer_legacy?
image

**Comment by brentyi** [comment]
let's move discussion to #1606!

**Pull request merged**